app/helpers/.gitignore

```diff
@@ -0,0 +1 @@
+!tags
```

Comment:
Is `/app/helpers/tags` not picked up by git for some reason?

Reply:
exactly, there is a global rule for `tags` probably for `ctags` (do not know if it necessary in all levels, not a ctags user :disappointed: